content/docs/atom-guide/dev-guide/materials/material-build-pipeline.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Material Build Pipeline
+draft: true
```

Comment:
I ran a search and found 3 links into this doc. To avoid broken links, you'll want to disable or remove those.